.github/actions/install-dependencies/action.yml

```diff
@@ -0,0 +1,10 @@
+name: 'Install dependencies'
+description: 'Install dependencies to build sssd'
+runs:
+  using: "composite"
+  steps:
+    - shell: bash
+      run: |
+          cd contrib/ci/
+          . deps.sh
+          deps_install
```

Comment:
Other places use 2-spaces for indentation (which I agree with), also use 2 spaces here intead of 4.